frontend/src/pages/MainPage.tsx

```
@@ -105,93 +105,96 @@ const MainPage = () => {
+            <span>Loading data</span>
+            <span class="inline-flex">
+              <span class="animate-bounce [animation-delay:-0.3s]">.</span>
+              <span class="animate-bounce [animation-delay:-0.15s]">.</span>
+              <span class="animate-bounce">.</span>
+            </span>
+          </div>
+          <Show when={reloadMutation.isPending}>
+            <button
+              onClick={handleStopReload}
+              class="mt-4 rounded-md border px-3 py-2"
+            >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)